lib/matplotlib/pyplot.py

```diff
@@ -15,7 +16,19 @@
     y = np.sin(x)
     plt.plot(x, y)
 
-The object-oriented API is recommended for more complex plots.
+The explicit (object-oriented) API is recommended for complex plots, though pyplot
+is still usually used to create the figure and often the axes in the figure.
+See `.pyplot.figure`, `.pyplot.subplots`, and `.pyplot.subplot_mosaic` to
+create figures, and :ref:`Axes API <../axes_api>` for the plotting
```

Comment:
> /home/circleci/project/lib/matplotlib/pyplot.py:docstring of matplotlib.pyplot:15: WARNING: undefined label: ../axes_api

This might be 

```suggestion
create figures, and :doc:`Axes API <../axes_api>` for the plotting
```
Not sure if the relative link works though.

Reply:
That worked.  Thanks!